src/components/pages/lessons/overlay/watch-next-lesson-cta-overlay.tsx

```diff
@@ -57,6 +61,39 @@ const WatchNextLessonCtaOverlay: React.FunctionComponent<{
           </a>
         </Link>
       </div>
+      {ctaContent && (
+        <div className="flex flex-col mt-6 md:mt-8 space-y-3">
+          <h3 className="text-md md:text-lg font-semibold mt-4 text-center">
+            {ctaContent.headline}
+          </h3>
+          {ctaContent.linksTo.map((content) => {
+            return (
+              <Link href={content.path || '#'}>
+                <a
+                  onClick={() => {
+                    track('clicked cta content', {
+                      from: lesson.slug,
+                      [content.type]: content.title,
+                    })
+                  }}
+                  className="px-3 py-2 flex items-center ml-4 transition-colors duration-200 ease-in-out"
+                >
+                  <div className="w-12 h-12 relative flex-shrink-0">
+                    <Image
+                      src={content.imageUrl}
+                      alt={`illustration of ${content.title} course`}
+                      width="64"
+                      height="64"
+                      layout="fill"
+                    />
+                  </div>
+                  {content.title}
+                </a>
+              </Link>
+            )
+          })}
+        </div>
+      )}
```

Comment:
from a visual design perspective this is NOT GOOD

![screenshot](https://p-zmfjnlq.b3.n0.cdn.getcloudapp.com/items/o0u7qkor/e1a7e531-ff6f-4249-9ec7-286793a3ec8f.jpg?v=d2b42cdd175ac985bd545a46b7467203)

Reply:
I did a [quick redesign](https://www.figma.com/file/mzc7h7IRA9Vy9pU4cgm9kE/Lesson-Overlay-CTA?node-id=0%3A1). This separates the two different actions that you could take. 'watch again' / 'play next' being the usual CTA while choosing another course being the additional CTA.

Also, the contrast ratio on the overlay is failing AAA contrast with a score of 2.17 (< AA+). Updating the background to `#555A60` gives us a score of ~7.0. 


![lesson overlay CTa](https://user-images.githubusercontent.com/6188161/139697499-fe6b301a-0acc-421a-95a8-d77873130b62.png)